docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,118 @@
+---
+lang: en
+title: 'Project Layout Reference'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Project-layout-reference.html
+---
+
+LoopBack 4 project files and directories are in the _application root
+directory_. Within this directory the standard LoopBack project structure has
+these sub-directories:
+
+- `src` - Node application scripts and configuration files.
+- `public` - Client JavaScript, HTML, and CSS files.
+- `definitions` - API and product definition YAML files (**IBM API Connect
+  only**).
+
+{% include tip.html content="By LoopBack naming conventions, artifacts such as models are grouped under their sub-directories. Also, with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case. For example, if you create a model named `MyUserModel` with the model generator, it creates files `my-user.model.ts` under `src/models`. See [Naming Convention](Command-line-interface.md#naming-convention) for more information.
+" %}
+
+<table style="font-size: 90%;">
+  <thead>
+    <tr>
+      <th width="200">File or directory</th>
+      <th>Description</th>
+      <th width="180">How to access in code</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">Top-level application directory</th>
+    </tr>
+    <tr>
+      <td><code>/node-modules</code> directory</td>
```

Comment:
I find it slightly weird that some entries start with a forward slash (`/node_modules`) while other don't (`package.json`). Let's be consistent please.

To differentiate between files (`package.json`) and directories (`node_modules`), I am proposing to _append_ a slash to directory names.

```suggestion
      <td><code>node-modules/</code> directory</td>
```